### PR TITLE
[@types/sortablejs](fix) Fix the MultiDragOptions' error

### PR DESCRIPTION
**Pull request by miyuesc**
Please fill in this template.

- [ x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ x] Test the change in your own code. (Compile and run.)
- [ x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [ x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

Changing an existing definition:
- [x ] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://github.com/SortableJS/Sortable/tree/master/plugins/MultiDrag>>

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/sortablejs/plugins.d.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 581e6d4df4f036359cfcf3f811ec176917274ca5
</p>

File: types/sortablejs/plugins.d.ts [hunk 1]
```diff
@@ -71,11 +71,12 @@ export interface MultiDragOptions {
...
-     * Key that must be down for items to be selected
+     * The key that must be down for multiple items to be selected.
+     * The default is null, meaning no key must be down.
+     * For special keys, such as the CTRL key,
+     * simply specify the option as 'CTRL' (casing does not matter).
      */
-    // todo: create a type
-    // todo: check source code for type
-    multiDragKey?: null | undefined;
+    multiDragKey?: null | string;
```
Comment (jakebailey) [inline types/sortablejs/plugins.d.ts:80]: You probably want to keep undefined, which is needed for exactOptionalPropertyTypes.

File: types/sortablejs/plugins.d.ts [hunk 2]
```diff
@@ -71,11 +71,12 @@ export interface MultiDragOptions {
...
```
Comment (codershiba) [inline types/sortablejs/plugins.d.ts:79]:
```suggestion
    multiDragKey?: null | undefined | string;
```

**Review by ghost — COMMENTED** [review]
Can you update tests?

**Review by RyanCavanaugh — APPROVED** [review]

**Comment by miyuesc** [comment]
Ready to merge

**Pull request merged**